Gemfile

```diff
@@ -8,7 +8,7 @@ gem 'ejson'
 gem 'fog'
 gem 'fog-json'
 gem 'fog-xml'
-gem 'fog-dynect'
+gem 'fog-dynect', :git => 'https://github.com/Shopify/fog-dynect.git'
```

Comment:
@es since by default, ALIAS records are not configured - I was under the assumption that it was a deliberate choice by the fog-dynect developers we were working around.

I've just created a PR in any case: https://github.com/fog/fog-dynect/pull/6